tex/chapters/literature.tex

```diff
@@ -1,11 +1,106 @@
 %!TEX root = ../main.tex
 
-\chapter{Awesome theorems and stuff}\label{cha:awesome_theorems_and_stuff}
+\chapter{Literature Review}\label{cha:literature_review}
 
-\section{Introduction}
+\section{Background}
 
-Note that I can refer to other chapters: see Chapter~\ref{cha:introduction} and
-even specific equations in each chapter, this is an
-equation~(\ref{equ:random_data}).
+- Explain Prisoner's Dilemma
 
-\lipsum
+- Axelrod's original tournament
+
+- Work to reproduce Axelrod's tournament ~\cite{Knight2016}
+
+\begin{tabular}{c c c c}\label{tab:tournament_refs}
+Year & Reference & Number of Strategies & Type\\
+\hline
+1979 & & 13 & Standard\\
+1979 & & 64 & Standard\\
+1984 & & 64 & Evolutionary\\
+1991 & & 13 & Noisy\\
+2005 & & 223 & Varied\\
+2012 & & 13 & Standard\\
+\hline
+% \caption{An overview of published tournaments}
+\end{tabular}
+
+is often used to model systems in biology ~\cite{Sigmund1999}, sociology ~\cite{Franken2005},
+psychology ~\cite{Ishibuchi2005}, and economics ~\cite{Chong2005}.
+
+
+\section{Fingerprinting}\label{sec:fingerprinting}
+
+\begin{definition}\label{def:joss-ann}
+If $A$ is a strategy for playing the iterated prisoner's dilemma, then the \textbf{Joss-Anne of A}, $JA(A, x, y)$ is a strategy which has a probablility $x$ of choosing move $C$, a probability $y$ of chosing move $D$, and otherwise uses the response appropriate to strategy $A$.
+\end{definition}
+
+The notation $JA$ comes from the initials of the names Joss and Anne.
+Joss was a strategy submitted to one of Axelrod’s original tournaments and it would occasionally defect without provocation in the hopes of a slight improvement in score.
+Anne is the first name of A. Stanley who suggested the addition of random cooperation (refs from ashlock paper) instead of random defection.
```

Comment:
Commenting to make sure "refs from ashlock paper" doesn't get missed.

Perhaps include `%TODO` somewhere near this line (you can then search through for `%TODO` at a later date).